github-checks-app/README.md

```diff
@@ -16,6 +16,8 @@
 5. Manually add the `webhook_secret` into the form, since setting webhook secret via query params is no longer supported 
 6. Scroll down to click `Create GitHub App`
    
-7. Once the app is created, you'll be redirected to the settings page of the app. Send us the `webhook_secret` from the URL above, as well as the `app_name`, `app_ID`, `client_ID`, `client_secret`, `private_key`:
+7. Once the app is created, pull down the app configuration from your app settings page (the app name, app ID, client ID, client secret, private key), as well as the webhook secret from the URL above
```

Comment:
Do you think a bulleted list might help?

Also, are they really pulling something down, or are they taking "note of"

Perhaps something like:
Once the app is created, from your app settings page, take note of the following values from the app of the configuration: 
* app name
* app ID
* client ID
* client secret
* private key

Also, you will need the webhook secret from the URL above